peps/pep-0728.rst

```diff
@@ -512,10 +510,11 @@ checks::
     movie: Movie = details  # Not OK. 'year' is not required in 'Movie',
```

Comment:
Doesn't the updated text below imply that this should now be "OK", rather than "not OK"?

Reply:
I think since "year" is not required in `Movie`, and required in `MovieWithYear`, `MovieWithYear` is not assignable to `Movie`, it's "not OK". Unless I'm missing something?

Maybe this can be rephrased as:

```python
    details: MovieWithYear = {"name": "Kill Bill Vol. 1", "year": 2003}
    movie: Movie = details  # Not OK. 'year' is not required in 'Movie',
                            # but it is required in 'MovieWithYear'
```

Reply:
Ah, makes sense now, thank you! It's a bit of a subtle rule: because we could mutate the dict (when we have it typed as `Movie`) to remove the `year` key, which would render a `MovieWithYear` invalid, we can't assign `MovieWithYear` to `Movie`.